src/main/scala/edu/cmu/cs/obsidian/parser/Parser.scala

```diff
@@ -212,23 +211,23 @@ object Parser extends Parsers {
 
         val parseUpdate = {
             val oneUpdate = parseId ~! EqT() ~! parseExpr ^^ {
-                case f ~ _ ~ e => (ReferenceIdentifier(f._1).setLoc(f), e)
+                case f ~ _ ~ e => (ReferenceIdentifier(f._1, None).setLoc(f), e)
             }
             LParenT() ~ repsep(oneUpdate, CommaT()) ~ RParenT() ^^ {
                 case _ ~ updates ~ _ => updates
             }
         }
 
         val parseTransition = RightArrowT() ~ parseId ~
-                              opt(parseUpdate) ~! SemicolonT() ^^ {
+            opt(parseUpdate) ~! SemicolonT() ^^ {
```

Comment:
This whitespace change isn't great.

Reply:
I agree. It's what IntelliJ did when I cleaned up the file and I think in line with what `scalafix` would do, but I'm not in love with it. I think a lot of linters don't quite know what to do with clever infixing.